src/pacer.js

```diff
@@ -236,14 +237,16 @@ let PACER = {
           }
         }
         // xxx does not match style above.
-        let match;
```

Comment:
Since you now assign to `match` twice, it seems like you'd be better off manually hoisting the `let match` up so that both assignments to `match` (at 240 and 246) looked similar, rather than having a `let` on the first one. YMMV I suppose?

Reply:
combining all three match usages under a single let match;